pypeit/specobj.py

```diff
@@ -350,39 +350,67 @@ def copy(self):
         # Return
         return copy.deepcopy(self)
 
-    def flexure_interp(self, sky_wave, fdict):
+    @staticmethod
+    def flexure_interp(shift, wave, wavein=None):
+        """
+        Perform interpolation on wave given a shift in pixels
+
+        Args:
+            shift (float):
+                Shift in pixels
+            wave (`numpy.ndarray`_):
+                extracted wave of size nspec
+            wavein (`numpy.ndarray`_, optional):
+                Apply the shift to this array of wavelengths
+        Returns:
+            `numpy.ndarray`_: Wavelength scale corrected for spectral flexure
+
+        """
+        npix = wave.size
+        x = np.linspace(0., 1., npix)
+        f = interpolate.interp1d(x, wave, bounds_error=False, fill_value="extrapolate")
+        twave = f(x + shift / (npix - 1))
+        # If this shift is to be applied onto another wave scale, reinterpolate
+        if wavein is not None:
```

Comment:
I thought we agreed to use the tilts image for this and not to use interp1d as you are doing it here.